src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,100 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int DOUBLE_CAPACITY = 2;
+    private static final int NEGATIVE_COEFFICIENT = -1;
+    private int size;
+    private final float threshold;
```

Comment:
Why is threshold _final_? We have to change this field in the code when we resize our map.